spec/mocks/mock-api.ts

```diff
@@ -0,0 +1,128 @@
+import { vi } from "vitest"
+import { jsonMockData } from "./mock-data"
+import { Product } from "../../src"
+import { Action, ActionResponse, Cart, Customer, Data, RenderMode } from "../../src/types"
+
+type LoadAction = {
+  load: () => Promise<Partial<ActionResponse>>
+}
+
+type PlacementAction = {
```

Comment:
Tried to use the `Action` type but it gets a little complex as nosto-react library doesn't use all the apis available in the Action type